taichi/gui/gui.h

```diff
@@ -503,7 +503,7 @@ class GUI : public GUIBase {
   };
 
   struct KeyEvent {
-    enum class Type { move, press, release };
+    enum class Type { move, press, release, scroll };
```

Comment:
is this used? Or is the plan to switch to `scroll` once wheel events are available on all OS?

Reply:
Typo, sorry!
```suggestion
    enum class Type { move, press, release };
```